python/tvm/relay/frontend/pytorch.py

```diff
@@ -1931,16 +1931,20 @@ def pad_common(self, mode, pad_value, inputs, input_types):
         for pad in paddings:
             const_paddings.append([])
             for p in pad:
-                if not isinstance(p, int):
+                if isinstance(p, _expr.Expr):
                     p = int(_infer_value(p, {}).numpy())
+                elif isinstance(p, float):
+                    p = int(p)
+                elif not isinstance(p, int):
```

Comment:
both float and int should work for `nn.pad`, casting to int here is not necessary

Reply:
good catch. This used to be a fix for converting any non-`Expr` pad value, aka float, into int. `_op.nn.pad` should be able to handle both `int` and `float`